assets/php/class-wgpb-block-library.php

```diff
@@ -639,6 +641,31 @@ public static function add_theme_body_class( $classes = array() ) {
 		$classes[] = 'theme-' . get_template();
 		return $classes;
 	}
+
+	/**
+	 * Update site options with post changes on publish/update.
+	 *
+	 * @param int     $post_id Post ID.
+	 * @param WP_Post $post    Post object.
+	 */
+	public static function save_data_to_settings( $post_id, $post ) {
+		if ( 'publish' !== $post->post_status ) {
+			return;
+		}
+
+		if ( has_block( 'woocommerce/checkout-privacy-policy', $post ) ) {
+			$blocks = wp_list_filter(
+				parse_blocks( $post->post_content ),
+				array( 'blockName' => 'woocommerce/checkout-privacy-policy' )
+			);
+			if ( empty( $blocks ) ) {
+				return;
+			}
+
+			$content = $blocks[0]['innerHTML'];
```

Comment:
This works well, but seems to encode `<br>` tags.
```
Your personal data will be used to process your order, support your experience throughout this website, and for other purposes described in our [privacy_policy].f&lt;br>&lt;br>
```

Other than that, line breaks seem to be working well in the editor for me.

Reply:
When saving in Gutenberg, I'm getting a new line before the text when I view in Customizer.  Probably not a big deal, but we can optionally trim this.

```suggestion
			$content = trim( $blocks[0]['innerHTML'] );
```